frontend/src/metabase/entities/users.js

```diff
@@ -24,13 +25,21 @@ function loadMemberships() {
   return require("metabase/admin/people/people").loadMemberships();
 }
 
+const getUserList = GET("/api/user");
+const getRecipientsList = GET("/api/user/recipients");
```

Comment:
I used this instead of `UserApi.list()` to be more clear about which API call is which